src/main/java/org/mule/extension/db/internal/domain/connection/mysql/MySqlConnectionParameters.java

```diff
@@ -33,7 +34,6 @@
   static final String NEW_MYSQL_DRIVER_CLASS = "com.mysql.cj.jdbc.Driver";
```

Comment:
these constants need to move to the factory

Reply:
done